manifest.yaml

```diff
@@ -292,15 +292,3 @@ remove-from-packages:
   # https://bugzilla.redhat.com/show_bug.cgi?id=1727058
   - - initscripts
     - "/"
-  # In newer dracut-network dhclient isn't required if you have NM 1.20
-  # https://github.com/dracutdevs/dracut/commit/e863807
-  # Let's go ahead and get rid of it now.
-  - - dhcp-client
-    # we want to delete all files ("/") but can't because
-    # `remove-from-packages` is being overzealous:
-    # https://github.com/coreos/rpm-ostree/issues/2068
-    - "/etc/dhcp/dhclient.conf"
-    - "/etc/NetworkManager/dispatcher.d/11-dhclient"
-    - "/usr/lib64/pm-utils/sleep.d/56dhclient"
```

Comment:
I think we want to keep these removed; the scope of the request is just to include the binaries themselves.  I think re-introducing `dhclient` and `dhclient-script` would be sufficient.